pypeit/core/wave.py

```diff
@@ -12,6 +12,11 @@
 from astropy.coordinates import UnitSphericalRepresentation, CartesianRepresentation
 from astropy.time import Time
 
+# THIS IS NEEDED TO RUN OFFLINE -- SHOULD IT ALWAYS BE SET?
+from astropy.utils.iers import conf
+conf.auto_max_age = None
```

Comment:
this is a long-standing and very open issue, https://github.com/astropy/astropy/issues/9227. this hack is _supposed_ to work, but in my experience is not always sufficient. i.e., if you have this set, but try to do a calculation that tries to interpolate the IERS tables, you get an error because it can't extrapolate beyond the end of the default tables provided by astropy. however, i don't think the code here does that so i think this workaround should be fine.